src/arm-elf-adac/arm-elf-adabind

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+# This file is part of the adac-hybrid-arm distribution
+# Copyright (C) 2022, European Space Agency
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, version 3.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+import sys
+import os
+
+
+class BexchData:
```

Comment:
I have a feeling that I wrote this code. This requires doc...

Reply:
Yes, this is your code.
Will try to provide docs.

Reply:
Done